node/pkg/node/node_test.go

```diff
@@ -841,7 +841,7 @@ func runConsensusTests(t *testing.T, testCases []testCase, numGuardians int) {
 	<-rootCtx.Done()
 	assert.NotEqual(t, rootCtx.Err(), context.DeadlineExceeded)
 	zapLogger.Info("Test root context cancelled, waiting 10ms for everything to shut down properly...")
-	time.Sleep(time.Millisecond * 10)
+	time.Sleep(time.Millisecond * 50)
```

Comment:
I think it would add value if we were to explain certain things happen outside of the `supervisor` lifecycle, ie databases close and log, and so do metrics sinks, which break the race detector.